components/lwip/port/esp32/include/lwipopts.h

```diff
@@ -697,37 +718,82 @@
 /**
  * ETHARP_DEBUG: Enable debugging in etharp.c.
  */
-#define ETHARP_DEBUG                    LWIP_DBG_OFF
+#ifdef CONFIG_LWIP_ETHARP_DEBUG
```

Comment:
These Kconfig changes to lwIP debug can be a separate PR that can be merged first, as they might be useful when implementing other network interfaces or debugging lwIP.

Reply:
thanks but, i'm not interested in splitting out or managing more prs at this time

Reply:
agree that this is a useful changeset, unrelated to the SLIP support. would suggest dissecting this change into a separate commit.